tests/integration/aof-multi-part.tcl

```diff
@@ -175,7 +175,7 @@ tags {"external:skip"} {
         }
 
         create_aof_manifest $aof_dirpath $aof_manifest_file {
-            append_to_manifest "filx appendonly.aof.1.base.aof seq 1 type b\n"
+            append_to_manifest "file_ appendonly.aof.1.base.aof seq 1 type b\n"
```

Comment:
I personally prefer to use `_` to "corrupt" a word to show that bad input isn't tolerated as opposed to actively misspelling words. Because entities make too many accidental typos (as shown in the rest of this PR), it seems best to avoid using such things as the test model.

I didn't apply this to all instances (I can't remember which ones I skipped -- this PR took a couple of days to develop...), as with everything else, I'm happy to drop.